helm/cluster-azure/templates/helmreleases/csi.yaml

```diff
@@ -21,7 +21,7 @@ spec:
       chart: azuredisk-csi-driver-app
       # used by renovate
       # repo: giantswarm/azuredisk-csi-driver-app
-      version: 1.27.0
+      version: 1.26.2-gs5
```

Comment:
why downgrade?

Reply:
@calvix the previous version was wrong. See https://gigantic.slack.com/archives/C02HLSDH3DZ/p1701160030457269?thread_ts=1701094751.226559&cid=C02HLSDH3DZ